roles/test_report_send/README.md

```
@@ -0,0 +1,610 @@
+* runtime metadata, either:
+    * `trs_metadata_path`
+    * CI-specific metadata collected from environment variables
+This event is sent to the collector (currently only splunk is supported).
+
+
+
+  - **trs_report_path**
+    - **Required**: False
+    - **Type**: str
+    - **Default**: 
+    - **Description**: Test report JSON file to send.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)